### PR TITLE
spring boot testing

### PR DESCRIPTION
**Pull request by BohdanZorii**

### CODE REVIEW

**Comment by BohdanZorii** [comment]
image

File: src/test/resources/db/changelog/db.changelog-master-test.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/changes/creation-scripts/01-create-books-related-tables.yaml
+  - include:
+      file: db/changelog/changes/creation-scripts/02-create-users-related-tables.yaml
+  - include:
+      file: db/changelog/changes/creation-scripts/09-create-shopping-cart-related-tables.yaml
+  - include:
+      file: db/changelog/changes/creation-scripts/12-create-order-related-tables.yaml
+
```
Comment (okuzan) [inline src/test/resources/db/changelog/db.changelog-master-test.yaml:10]: 4 extra lines

File: src/test/resources/database/delete-all-books-and-categories.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+DELETE FROM books
+
+DELETE FROM categories
```
Comment (okuzan) [inline src/test/resources/database/delete-all-books-and-categories.sql:3]: please always leave last line empty
image

File: src/test/resources/application.properties [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
+spring.datasource.password=test
 
-jwt.expiration=300000
+jwt.expiration=30000000
 jwt.secret=pleaseDontReadThisIsVerySecretString
+
+spring.liquibase.change-log=classpath:/db/changelog/db.changelog-master-test.yaml
+
+
+
```
Comment (okuzan) [inline src/test/resources/application.properties:11]:
```suggestion
```

File: src/test/java/mate/zorii/bookstore/controller/BookControllerTests.java [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+                .build();
+    }
+
+    @WithMockUser(username = "user", roles = {"USER"})
+    @Test
+    @DisplayName("Get all books with pagination")
+    @Sql(scripts = "classpath:database/insert-3-books.sql",
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = "classpath:database/delete-all-books-and-categories.sql",
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
```
Comment (okuzan) [inline src/test/java/mate/zorii/bookstore/controller/BookControllerTests.java:57]: you can extract it to a class level, if it's used on (almost) every method

**Review by okuzan — APPROVED** [review]
Good job!

**Pull request merged**